### PR TITLE
Backend api post

### PR DESCRIPTION
**Pull request by mgomez05**
Added some readme information for creating and deleting databases, and some helpful code for serializing objects and using POST parameters

But more importantly, finished the post API method, which now creates a new user if necessary when posting a textbook, and saves it into the database

P.S. You're all lucky I had an ethernet cable

### CODE REVIEW

**Comment by bkcapretta** [comment]
Make sure to pull from master and fix those merge conflicts - you have six conflicting files

File: backend/api/models.py [hunk 1]
```diff
@@ -1,18 +1,29 @@
 from django.db import models
-
 # Two tables for scalability purposes
+
+#User Table:
+class user(models.Model):
+    #user_id = models.PositiveIntegerField(primary_key=True)
```
Comment (bkcapretta) [inline backend/api/models.py]: Why is this one commented out?

File: backend/api/views.py [hunk 2]
```diff
@@ -1,14 +1,77 @@
...
 
 # Create views here!
 class ListBookRaw(generics.ListCreateAPIView):
     queryset = Textbook.objects.all()
     serializer_class = SellSerializer
 
+    def post(self, request, *args, **kwargs):
+
+        # Loop through post parameters
+        #for key in request.POST:
```
Comment (bkcapretta) [inline backend/api/views.py]: Do you need this or could you remove these personal comments?

**Comment by mgomez05** [comment]
> Make sure to pull from master and fix those merge conflicts - you have six conflicting files

**Comment by mgomez05** [comment]
Okay, I just pulled from origin master

File: backend/api/views.py (hunk 2 above)
Comment (mgomez05) [reply backend/api/views.py]: I have now removed the personal statements  and placed the useful ones in the README

File: backend/api/models.py (hunk 1 above)
Comment (mgomez05) [reply backend/api/models.py]: I don't know, Juliet set up the user class, I just built off of her code

**Pull request merged**